src/test/java/reposense/git/GitLogTest.java

```diff
@@ -22,14 +22,14 @@ public void gitLog_existingFormats_hasContent() {
 
     @Test
     public void gitLog_nonExistingFormats_noContent() {
-        config.setFormats(Collections.singletonList(new Format("py")));
+        config.setFormats(FileType.convertStringFormatsToFileTypes(Collections.singletonList("py")));
         String content = GitLog.get(config, getAlphaAllAliasAuthor());
         Assert.assertTrue(content.isEmpty());
     }
 
     @Test
     public void gitLog_includeAllJavaFiles_success() {
-        config.setFormats(Collections.singletonList(new Format("java")));
+        config.setFormats(FileType.convertStringFormatsToFileTypes(Collections.singletonList("java")));
```

Comment:
See [here](https://github.com/reposense/RepoSense/pull/787/files#diff-37415d1fc78bc1091efc3c5eec01d822R11), I believe better to rename the method to `convertFormatStringsToFileTypes`